### PR TITLE
Applicants tabbed navigation

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds tabbed navigation for `/applicants` pages

## Description

- Adds the `Tabs` component to `ApplicantsLayout`
- Modifies the `ApplicantsLayout` component to persist state of current tab. This is needed to show the bottom accent on selected tab.
- It sets `tabIndex` using `router.pathname` on initial page load.* 

*The reason for this is that we need the current tab number 
  1. before the component loads
  2. on tab change
  3. when loading a certain `/applicants` page, i.e: when navigating directly to the `/applicants/office-hours` url, current tab should be set to 1, but there's no previous event that sets the initial value of current tab to 1.

For this reasons, using Context to provide a global state (original approach) was discarded and the tab is set based on pathname.
 
 PREVIEW: https://deploy-preview-123--thirsty-meitner-cae269.netlify.app/applicants

### CODE REVIEW

File: src/components/UI/common/Footer.tsx [hunk 1]
```diff
@@ -31,12 +33,12 @@ export const Footer: FC = () => {
               Home
             </ImportantText>
           </Link>
-          <Link href='/applicants' _hover={{ textDecoration: 'none' }}>
+          <Link href={APPLICANTS_URL} _hover={{ textDecoration: 'none' }}>
```
Comment (samajammin) [inline src/components/UI/common/Footer.tsx:36]: I like this approach 👍

File: src/components/layout/ApplicantsLayout.tsx [hunk 2]
```diff
@@ -1,27 +1,128 @@
...
+export const ApplicantsLayout: FC = ({ children }) => {
+  const router = useRouter();
+  const [tabIndex, setTabIndex] = useState(TABS_MAP[router.pathname]);
+
+  const handleChange = (index: number) => {
+    setTabIndex(index);
+
+    switch (index) {
+      case 0:
+        router.push(
```
Comment (samajammin) [inline src/components/layout/ApplicantsLayout.tsx:26]: As I mentioned on that call, I think it'd be ideal for these tabs to actually be links given that we replicate the same functionality (i.e. clicking a tab routes the user to that route).

Is this something the `Tabs` Chakra component can support? Or would this require significant changes?

In any case, I see this as a "nice to have", not a blocker.

**Review by samajammin — APPROVED** [review]
Nice work!

File: src/components/layout/ApplicantsLayout.tsx (hunk 2 above)
Comment (nhsz) [reply src/components/layout/ApplicantsLayout.tsx:26]: Yes, it's supported, rendering the tab as an `<a>` and adding cursor pointer.

Screen Shot 2022-01-10 at 16 06 39

Screen Shot 2022-01-10 at 16 03 04

**Pull request merged**